.changesets/fix_fix_coprocessor_data_null.md

```diff
@@ -0,0 +1,9 @@
+### Fix Parsing of Coprocessor GraphQL Responses ([PR #7141](https://github.com/apollographql/router/pull/7141))
+
+Standardized GraphQL response parsing and validation between coprocessors and subgraphs to ensure consistent behavior throughout the Router. 
```

Comment:
I think we need a user focused changset entry. Something like.

Previously coprocessors allowed null body in graphql responses, hoever this is not allowed in the graphql spec. The router now correctly enforces that the body of a coprocessor response is valid, and will reject such responses.

If your coprocessor relied on this behaviour then you must update your coprocessor to return a calid graphql response.

Reply:
Make sense 👍